### DISCUSSION TITLE
Empty Y value but still print the X axis ?

### DISCUSSION THREAD

**Discussion by neobie** [patrykandpatrick/vico#843] [Questions]
### Question

![image](https://github.com/user-attachments/assets/0f8273cf-817c-46a4-be27-168c73e68df9)

This is example taken from Swift WatchOS chart.
It can render the chart X axis (until 5pm), but not to render the Y value when it is still empty (eg 0)

Similar question: https://stackoverflow.com/questions/62716141/linechart-avoid-plotting-of-0-values

How can I do that ?

### Vico version(s)

1.15

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
You can use `AxisValuesOverrider` to do this. Please see, for example, [here](https://github.com/patrykandpatrick/vico/discussions/793#discussioncomment-10156281).
